Directory.Build.targets

```diff
@@ -75,90 +75,5 @@
   <Import Project="eng\targets\Wix.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.wixproj'" />
   <Import Project="eng\targets\Npm.Common.targets"  Condition="'$(MSBuildProjectExtension)' == '.npmproj'" />
 
-  <!-- Prepares the test projects for helix by including xunit and publishing -->
```

Comment:
+1 for moving all this helix stuff to another file. I was going to do this anyways, so thanks for taking care of it.